homework/content.js

```diff
@@ -1,37 +1,40 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
-
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
-
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
+(function(exports) {
+	var ContentManager = function () {
+	this._wrapper = document.querySelector('#note-content-wrapper'}
+  
   }
 
-  function drawNote(note) {
-    var title = note.title;
-    var h = document.createElement('h2');
-    h.textContent = title;
-    var passages = note.passages;
-    var buff = document.createDocumentFragment();
-    passages.forEach(function(passage) {
-      var p = document.createElement('p');
-      p.classList.add('note-passage');
-      p.textContent = passage;
-      buff.appendChild(p);
-    });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
-  }
+  ContentManager.prototype = {
+	start(){
+		window.addEventListener('note-open', (function(event){
```

Comment:
Instead of binding an anonymous function, you should bind the `this` as the listener. And Arrow function is better than the binding API (it's more concise), although I didn't require you to use that. See:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Lexical_this